src/staging/src/pages/park-search.js

```diff
@@ -247,187 +254,14 @@ export default function Home({ location, data }) {
-    //   filterOptions.push({ all: [{ typecode: ["ER"] }] })
-    // }
-
-    // if (parkActivitiesFilter && parkActivitiesFilter.length > 0) {
-    //   filterOptions.push({ all: [...parkActivitiesFilter] })
-    // }
-
-    // if (parkFacilitiesFilter && parkFacilitiesFilter.length > 0) {
-    //   filterOptions.push({ all: [...parkFacilitiesFilter] })
-    // }
-
-    // const options = {
-    //   search_fields: {
-    //     protectedareaname: {},
-    //     parkactivities: {},
-    //     parkfacilities: {},
-    //   },
-    //   filters: {
-    //     all: filterOptions,
-    //   },
-    //   result_fields: {
-    //     typecode: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //     protectedareaname: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //     marineprotectedarea: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //     parkactivities: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //     url: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //     parkfacilities: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //     isdayusepass: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //     parkadvisories: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //     parkphotos: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //     opentopublic: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //     slug: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //     id: {
-    //       raw: {},
-    //       snippet: { fallback: true },
-    //     },
-    //   },
-    //   page: { size: itemsPerPage, current: resetCurrentPage ? 1 : currentPage },
-    // }
-    // if (sortOption && sortOption.value && sortOption.value !== "rel") {
-    //   options.sort = { protectedareaname: sortOption.value }
-    // }
-
-    const results = searchParkByCriteria(
-      false,
-      protectedAreas,
-      selectedActivities,
-      selectedFacilities,
-      searchText,
-      quickSearch.camping,
-      quickSearch.petFriendly,
-      quickSearch.wheelchair,
-      quickSearch.marine,
-      quickSearch.ecoReserve,
-      quickSearch.electricalHookup
-    )
-    if (sortOption.value === "asc") {
-      results.sort(sortAsc)
-    } else {
-      results.sort(sortDesc)
-    }
+  function processResults(results) {
+    console.log("DATA:", results);
     const allResults = results.map(r => ({
       protectedAreaName: r.protectedAreaName,
       isOpenToPublic: true,
       advisories: ["Wildfire alert"],
       isDayUsePass: true,
-      parkActivities: r.parkActivities.map(a => a.name.split(":")[1]),
```

Comment:
park activities and facilities missing in search results page.

Reply:
Also, sorting doesn't seem to be working on park-search page although I can see that the search-view results are sorted appropriately.

Reply:
Please correct me if I am wrong, but I can see that the park photos and advisories are still hard coded. My understanding was that the MariaDB search query would be pulling in the data from appropriate content types in Strapi.

Reply:
Fixed in ec222d0

Reply:
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'map')
    at eval (webpack-internal:///./src/pages/park-search.js:280)
    at Array.map (<anonymous>)
    at processResults (webpack-internal:///./src/pages/park-search.js:273)
    at eval (webpack-internal:///./src/pages/park-search.js:321)

Getting this error with the latest fix

Reply:
This is only an iteration, there is more work to do after this lands.  I'll back out ec222d0 in favour of fixing it later.

Reply:
Reverted in 2751960